src/ai/backend/manager/api/session.py

```diff
@@ -85,6 +86,7 @@
 )
 from ai.backend.common.plugin.monitor import GAUGE
 
+from ..background import ProgressReporter
```

Comment:
Tip: This is not a necessary fix, but just FYI. When you don't instantiate a referred type directly but only use it as an annotation, you may put the import inside `if TYPE_CHECKING:` block below, for slightly faster module loads and avoid circular imports by improving independency of each module.